frontend/src/wrappers/formatDate.js

```diff
@@ -0,0 +1,13 @@
+export const formatDate = (fullDate) => {
+  var today = new Date(fullDate);
+
+  var date =
+    today.getDate() + '.' + (today.getMonth() + 1) + '.' + today.getFullYear();
+
+  var time = today.getHours() + ':' + today.getMinutes();
+  //  + ':' + today.getSeconds();
```

Comment:
Komentarz do usunięcia

Reply:
Tutaj komentarz jest celowy, ale nie wyjaśniłem tego w wystarczający sposób. Jeżeli zdecydujemy się, żeby data pokazywała również sekundy, to wystarczy przywrócić zakomentowany fragment.

Reply:
nope, wystarczy to dopisać, to nie jest wiedza tajemna, a komentarz będzie zaśmiecać tylko kod i czytelność